dbms/sqlquery/mysql/src/cloud/benchflow/monitors/querymysql.go

```diff
@@ -35,35 +36,40 @@ func queryHandler(w http.ResponseWriter, r *http.Request) {
     }
     // Checks over the rows if the comparison holds
     rowI := 0
```

Comment:
@VincenzoFerme turns out I don't, I had it in case I needed the row index. Removed it.